sites/public/src/patterns/Hero.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+import { Heading } from "@bloom-housing/ui-seeds"
+import MaxWidthLayout from "../layouts/max-width"
+import styles from "./Hero.module.scss"
+
+export interface HeroProps {
+  /** A clear call to action, most typically a button */
+  action: React.ReactNode
+  /** Optional text shown above the heading */
+  note?: string
+  /** Text shown beneath the heading */
+  subtitle?: string
+  /** Main heading text */
+  title: string
```

Comment:
nit: typically required fields should be before optional fields

Reply:
no worries!